packages/@okta/vuepress-site/docs/release-notes/2022-okta-identity-engine/index.md

```diff
@@ -5,6 +5,18 @@ title: Okta Identity Engine API Products release notes 2022
 
 ## March
 
+### Weekly release 2022.03.2
+
+| Change | Expected in Preview Orgs |
+|--------------------------------------------------------------------------|--------------------------|
+| [Bugs fixed in 2022.03.2](#bugs-fixed-in-2022-03-2) | March 16, 2022 |
+
+#### Bugs fixed in 2022.03.2
+
+* An error was incorrectly returned when a valid [update request](/docs/reference/api/authorization-servers/#update-a-scope) was made for the `device_sso` or `online_access` system scopes. (OKTA-417477)
```

Comment:
I notice that in past release notes we haven't used the word "incorrectly" in bug blurbs. Isn't it implied because it's a bug?
```suggestion
* An error was returned when a valid [update request](/docs/reference/api/authorization-servers/#update-a-scope) was made for the `device_sso` or `online_access` system scopes. (OKTA-417477)
```

Reply:
"incorrectly" was added in the RN hint and was approved by RN editor and dev mgr. You're right though - that "incorrectly" is implied since it's a bug, I didn't think to remove it. Let me confirm with Susan/Graham before removing.